models/blobs/ethereum/blobs_ethereum_schema.yml

```diff
@@ -0,0 +1,69 @@
+version: 2
+
+models:
+  - name: transactions
+    meta:
+      blockchain: ethereum
+      sector: blobs
+      project: blobs
+      contributors: [msilb7, 0xrob, lorenz234]
+    config:
+      tags: ['blob', 'blobs', 'BLOB_TX_TYPE']
+    description: Combines blobs data with their corresponding L1 transaction data
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - blob_versioned_hash
+    columns:
+      - &block_time
+        name: block_time
+        description: "UTC event block time of the transaction"
+      - &block_number
+        name: block_number
+        description: "Block number of the transaction"
+      - &tx_hash
+        name: tx_hash
+        description: "hash of the L1 transaction"
+      - &tx_from
+        name: tx_from
+        description: "address of the sender"
+      - &tx_to
+        name: tx_to
+        description: "address of the receiver"
+      - &tx_gas_used
+        name: tx_gas_used
+        description: "gas units used by L1 transaction"
+      - &tx_l1_gas_price
+        name: tx_l1_gas_price
+        description: "gas price of the L1 transaction"
+      - &tx_l1_base_fee
+        name: tx_l1_base_fee
+        description: "base fee of the L1 transaction"
+      - &blob_versioned_hashes
+        name: blob_versioned_hashes
+        description: "array of blob hashes for one L1 transaction"
+      - &blob_versioned_hash
+        name: blob_versioned_hash
+        description: "hash of the blob"
+      - &used_blob_byte_count
+        name: used_blob_byte_count
+        description: "byte size of the blob"
+      - &blobgas_used_by_blob
+        name: blobgas_used_by_blob
+        description: "gas units used by blob"
+      - &max_fee_per_blob_gas
+        name: max_fee_per_blob_gas
+        description: "max fee per blob gas of the blob"
+      - &excess_blob_gas
+        name: excess_blob_gas
+        description: "excess blob gas of the blob"
+      - &blob_base_fee_per_gas
+        name: blob_base_fee_per_gas
+        description: "gas price of the blob"
+      - &unique_id
```

Comment:
We can maybe cut this from the spell, since it's just row_number and it won't hold over incremental builds